### ISSUE TITLE
traffic_light_state/future in training dataset for Motion Prediction Challenge

### ISSUE THREAD

**Issue by YUQIJLU** [waymo-research/waymo-open-dataset#696]
Hi,

May I ask if the traffic light states of the next 8 seconds are provided in the training dataset? I noticed the description of these data when reading the details of tf.Example Format, but I could not find them when parsing the dataset.

Thanks
Yuqi
![6aa23a325984f41617eeb140171905d](https://github.com/waymo-research/waymo-open-dataset/assets/120549113/86d7949e-939c-4b70-bd8b-039b92ca7cdd)

**Comment by scott-ettinger** [comment]
Yes, these fields exist in both the training and validation splits.  The test split does not contain future data.  I just verified this.  How are you parsing the data?  Maybe those fields are not getting included?

**Comment by liuscot** [comment]
I meet the same thing too, please let me know how to get the traffic light states of the next 8 seconds? thx
